appveyor.yml

```diff
@@ -2,37 +2,44 @@
 #     DEVELOPMENT BRANCH
 #
 # build in Debug mode and deploy to MyGet.org as pre-release
-- version: 1.4.1-develop+{build}
-  branches:
+
+- branches:
     only:
     - develop
+
   configuration: Debug
   platform: Any CPU
+
   environment:
     use_build_number_in_packages: true
-  pull_requests:
-    do_not_increment_build_number: true
+
+  artifacts:
+    - path: src\build\Debug
+    - path: src\build\nuget\*.nupkg
+
   cache:
-  - src\packages -> src\**\packages.config
-  install:
-  - ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/RadicalFx/Radical.Build/master/appveyor-buildversion.ps1'))
-  assembly_info:
-    patch: true
-    file: '**\AssemblyInfo.*'
-    assembly_version: '$(assembly_version)'
-    assembly_file_version: '$(assembly_file_version)'
-    assembly_informational_version: '$(assembly_informational_version)'
+    - src\packages -> src\**\packages.config
+
   before_build:
-  - nuget restore "src\Radical.Windows.Presentation.CastleWindsor.sln"
+    - nuget restore "src\Radical.Windows.Presentation.CastleWindsor.sln"
+
   build:
     project: src\Radical.Windows.Presentation.CastleWindsor.sln
     verbosity: minimal
+
   after_build:
-  - md "src\build\nuget"
-  - nuget pack "src\nuget\Radical.Windows.Presentation.CastleWindsor.nuspec" -Version "%package_version%" -Properties "configuration=%CONFIGURATION%" -OutputDirectory "src\build\nuget"
-  artifacts:
-  - path: src\build\Debug
-  - path: src\build\nuget\*.nupkg
+    - md "src\build\nuget"
+    - nuget pack "src\nuget\Radical.Windows.Presentation.CastleWindsor.nuspec" -Version "%GitVersion_NuGetVersion%" -Properties "configuration=%CONFIGURATION%" -OutputDirectory "src\build\nuget"
+
+  pull_requests:
+    do_not_increment_build_number: true
```

Comment:
for this point I'm a little bit confused about the version is generating right now:

`1.5.0-PullRequest.14+6.build.40`

I see a `+6` and `build.40`, so who is adding that? what if we removed this `do_not_increment_build_number`, that was a feature to simply avoid incrementing appveyor build number on PR, but probably now is useless because gitversion seems to take ownership also of the appveyor version.